src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -431,10 +467,19 @@ trait WorkflowComponent {
       filter(_.id inSetBind(ids))
     }
 
+    def findWorkflowByExternalId(externalId: String): WorkflowQueryType = {
```

Comment:
this function should also go away, it is only used in tests and should probably be replaced by findWorkflowByExternalIdAndSubmissionId